scripts/run_param_parse.py

```diff
@@ -92,14 +92,16 @@ def run(self):
             except Exception as exception:
                 print(exception)
 
-            output = f'/tmp/output'
+            output = '.'
```

Comment:
To be safe, this should use:
```suggestion
            output = Path(__file__).parent.parent
```